odom/odom/config/odom_global.yaml

```diff
@@ -0,0 +1,52 @@
+frequency: 50
+
+# Specify planar environment
+two_d_mode: true
+
+map_frame: caffeine/map
+odom_frame: caffeine/odom
+base_link_frame: caffeine/base_link
+world_frame: caffeine/map
+
+# publish transform: world_frame --> odom_frame
+publish_tf: true
+
+transform_time_offset: 0.05
+
+# See /diagnostics topic for debug
+print_diagnostics: false
+
+## Odometry Sensor(s) ##
+
+odom0: caffeine/odometry/gps
+odom0_config: [true,  true,  false,
+               false, false, false,
```

Comment:
should yaw be true here?

Reply:
no since the gps odom (navsat) does not publish yaw (ie. it never changes for odom/gps)